src/SignalR/samples/SignalRSamples/wwwroot/hubs.html

```diff
@@ -83,6 +84,13 @@ <h4>Group Actions</h4>
 <script src="lib/signalr/signalr-protocol-msgpack.js"></script>
 <script src="utils.js"></script>
 <script>
+    var name = prompt("Please enter your name", "D'Artagnan");
```

Comment:
Can we just have a normal text input box with a default value of "Anonymous" or "D'Artagnan" or whatever for the user name? That way it doesn't block the entire browser UI.

Reply:
Agreed. We had the `prompt` in other places where we connected on page load. If we have a "Connect" gesture we can use a standard text box. I would also suggest having the "Connect" button check for a non-empty user name and produce an error (and not connect) in that case.